### PR TITLE
Blazor VSC launch/tasks/debug guidance updates

### LINKED ISSUE #22234: Missing Guidance on VSCode setup for SignalR with Blazor
In the step "When the dialog appears to add assets to build and debug the app, select Yes. Visual Studio Code automatically adds the .vscode folder with generated launch.json and tasks.json files."

When you press "OK", vscode asks whether you want to set it up for Server or Client. You have to choose Server or you get errors like:
```
command 'blazorwasm-companion.launchDebugProxy' not found
```
and things partially break.
It took me a while to figure out where I went wrong.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 097456af-6293-8c24-3991-fc815b94dcbf
* Version Independent ID: 978264ac-18f8-e937-30db-7050e53ac7d1
* Content: [Use ASP.NET Core SignalR with Blazor](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr-blazor?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/signalr-blazor.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/signalr-blazor.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #22234

Thanks @samcarey! #22325 

Notes:

* It's challenging to cover cleanly because there are permutations of hosting (standalone vs. hosted), some sort of 🐞 *bug* in VSC `launch.json`/`task.json` automatic config, stock launch/task setup (non-debugging) vs. launch/task setup with debugging.
* I think what I've worked out covers things, and we can take feedback on this to see if further confusion 😵 ensues.
* Because I favor adding the `.vscode` assets to a hosted WASM app's **_solution folder_**, I don't think we should keep the old *Launch and debug a hosted Blazor WebAssembly app with Microsoft Edge* section, so I delete it here. That's now covered by the new content added on this PR. By opening at the solution level, all of the solutions folders/projects (`Client`, `Server`, and `Shared`) are visible to the user. When run from the solution's folder, it runs the `Server` project. That's what I prefer to do anyway in my own use of VSC with hosted WASM.
* My local test of this guidance was successful using the Blazor-SignalR tutorial hosted WASM solution. I 🙈 *RexHacked!*&trade; 🙈 my way to the coverage on this PR. I'm able to set and hit breakpoints client- and server-side. However, I'm a little concerned about this being correct and stable. 👀👂 
* I haven't reached the Blazor WASM Debugging topic yet on the [UE passes](https://github.com/dotnet/AspNetCore.Docs/issues/19286), so I'll get another shot at updates when it comes up for general work.
* Disregard the build failure: It's a false positive for this PR. We'll merge past it after review+updates+approval.

### CODE REVIEW

**Comment by guardrex** [comment]
@Rick-Anderson ... I thought you killed off the *Cyberdine Systems Snippet Builder 5000*&trade;???

terminator

&mdash;  **_I'm back._** Cyberdyne Systems T-800 Series Model 101 ([Arnold Schwarzenegger](https://www.imdb.com/name/nm0000216/))
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[*Terminator 3: The Rise of the Machines*](https://warnerbros.fandom.com/wiki/Terminator_3:_Rise_of_the_Machines) ©2003 [Skydance Media](https://skydance.com/)

**Comment by Rick-Anderson** [comment]
@guardrex  I'll be back
That's something different and unrelated.

**Comment by guardrex** [comment]
Oh, I see. I thought those two were related. This is just a failure on warnings but that leave the PR mergable. Got it.

File: aspnetcore/blazor/debug.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Visual Studio for Mac requires version 8.8 (build 1532) or later:
 > [!NOTE]
 > Apple Safari on macOS isn't currently supported.
 
-## Enable debugging
+## Debug an existing standalone Blazor WebAssembly app
```
Comment (captainsafia) [inline aspnetcore/blazor/debug.md:60]:
```suggestion
## Debug a standalone Blazor WebAssembly app
```

I don't think the "existing" is necessary here?

**Review by captainsafia — APPROVED** [review]
Looks good!

Re: the other thread. There's been some churn in the VS Code extension recently so I'll try and repro some of this stuff. We've been changing some of the codepaths related to detecting Blazor projects.

**Pull request merged**